### PR TITLE
add highlights for 2023-10-12

### PR DESCRIPTION
**Pull request by weatherhog**
This weeks highlights PR. Please add your contributions till Wednesday 2023-10-11 eob.

### CODE REVIEW

File: src/content/changes/highlights/2023-10-12.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-product
+---
+
+## General
+
+## Apps
+
+## Documentation
+
```
Comment (stone-z) [inline src/content/changes/highlights/2023-10-12.md:15]:
```suggestion
## Documentation

Migration away from Pod Security Policies has begun! Please review the new [cluster administrator migration guide](https://docs.giantswarm.io/advanced/security-policy-enforcement/cluster-admin-guide/) to learn about the new [Policy API](https://docs.giantswarm.io/advanced/security-policy-enforcement/policy-api/) and the assistive tooling available to help you securely migrate workloads off of PSPs.

```

File: src/content/changes/highlights/2023-10-12.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  - Highlights
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-product
+---
+
+## General
+
+## Apps
+
+## Documentation
```
Comment (stone-z) [inline src/content/changes/highlights/2023-10-12.md:14]:
```suggestion
## Apps

- [security-bundle](https://github.com/giantswarm/security-bundle) versions [1.1.0](https://github.com/giantswarm/security-bundle/blob/main/CHANGELOG.md#110---2023-10-10) and [0.18.0](https://github.com/giantswarm/security-bundle/blob/legacy/CHANGELOG.md#0180---2023-10-05) include two new tools supporting migration away from Pod Security Policies: [`exception-recommender`](https://github.com/giantswarm/exception-recommender) analyzes the current policy reports in a cluster and generates Giant Swarm `PolicyExceptionDrafts`. Once the drafts have been reviewed and accepted, [`kyverno-policy-operator`](https://github.com/giantswarm/kyverno-policy-operator/) takes the resulting Giant Swarm `PolicyExceptions` and generates the necessary Kyverno resources to allow workloads to continue running.

## Documentation
```

File: src/content/changes/highlights/2023-10-12.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+changes_categories:
+  - Highlights
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-product
+---
+## Apps
+
+- [security-bundle](https://github.com/giantswarm/security-bundle) versions [1.1.0](https://github.com/giantswarm/security-bundle/blob/main/CHANGELOG.md#110---2023-10-10) and [0.18.0](https://github.com/giantswarm/security-bundle/blob/legacy/CHANGELOG.md#0180---2023-10-05) With these two releases we include two new tools supporting migration away from Pod Security Policies, [`exception-recommender`](https://github.com/giantswarm/exception-recommender) and [`kyverno-policy-operator`](https://github.com/giantswarm/kyverno-policy-operator/). With [`exception-recommender`](https://github.com/giantswarm/exception-recommender) analyzes the current policy reports in a cluster get analyzed and based on the results a Giant Swarm `PolicyExceptionDrafts` get generated. Once the drafts have been reviewed and accepted, [`kyverno-policy-operator`](https://github.com/giantswarm/kyverno-policy-operator/) takes the resulting Giant Swarm `PolicyExceptions` and generates the necessary Kyverno resources to allow workloads to continue running.
```
Comment (pipo02mix) [inline src/content/changes/highlights/2023-10-12.md:11]:
```suggestion
- [security-bundle](https://github.com/giantswarm/security-bundle) versions [1.1.0](https://github.com/giantswarm/security-bundle/blob/main/CHANGELOG.md#110---2023-10-10) and [0.18.0](https://github.com/giantswarm/security-bundle/blob/legacy/CHANGELOG.md#0180---2023-10-05) With these two releases we include two new tools supporting migration away from Pod Security Policies, [`exception-recommender`](https://github.com/giantswarm/exception-recommender) and [`kyverno-policy-operator`](https://github.com/giantswarm/kyverno-policy-operator/). With [`exception-recommender`](https://github.com/giantswarm/exception-recommender) analyzes the current policy reports in a cluster get analyzed and based on the results a Giant Swarm `PolicyExceptionDrafts` gets generated. Once the drafts have been reviewed and accepted, [`kyverno-policy-operator`](https://github.com/giantswarm/kyverno-policy-operator/) takes the resulting Giant Swarm `PolicyExceptions` and generates the necessary Kyverno resources to allow workloads to continue running.
```

File: src/content/changes/highlights/2023-10-12.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+## Documentation
+
+We have started the migration away from Pod Security Policies! Therefore we have added a [cluster administrator migration guide](https://docs.giantswarm.io/advanced/security-policy-enforcement/cluster-admin-guide/) containing all information about the new [Policy API](https://docs.giantswarm.io/advanced/security-policy-enforcement/policy-api/) and all the assistive tooling available to help you securely migrate workloads off of PSPs. Reach out for anyquestions regarding the Pod Security Policies to Pod Security Stadards migration
```
Comment (pipo02mix) [inline src/content/changes/highlights/2023-10-12.md:15]:
```suggestion
We have started the migration away from Pod Security Policies! Therefore we have added a [cluster administrator migration guide](https://docs.giantswarm.io/advanced/security-policy-enforcement/cluster-admin-guide/) containing all information about the new [Policy API](https://docs.giantswarm.io/advanced/security-policy-enforcement/policy-api/) and all the assistive tooling available to help you securely migrate workloads off of PSPs. Reach out for any questions regarding the Pod Security Policies to Pod Security Standards migration
```

**Review by pipo02mix — APPROVED** [review]
just minor suggestions but LGTM

**Pull request merged**